Roadmap.md

```diff
@@ -33,7 +36,44 @@ Tasks are tracked on a weekly basis through a dedicated spreadsheet.
 For more information see [Confidential Containers V0 Plan](https://docs.google.com/spreadsheets/d/1M_MijAutym4hMg8KtIye1jIDAUMUWsFCri9nq4dqGvA/edit#gid=0&fvid=1397558749).
 
 
-# Mid/long term roadmap 
+# Mid term roadmap
+
+Continue our journey using knowledge and support of SME's in other projects to form stronger 
```

Comment:
SME ?

Reply:
SME = subject matter experts. 
James, I'd go with the full name :-)

Reply:
yup will do

Reply:
done